modules/storage-page-memory/src/test/java/org/apache/ignite/internal/storage/pagememory/VolatilePageMemoryMvTableStorageTest.java

```diff
@@ -60,8 +58,6 @@
 public class VolatilePageMemoryMvTableStorageTest extends AbstractMvTableStorageTest {
     private final PageEvictionTracker pageEvictionTracker = spy(PageEvictionTrackerNoOp.INSTANCE);
```

Comment:
Just a small nitpick: what's the point of using "spy" here, while a simple mock should be enough?

Reply:
I want to use an implementation provided out of the box so as not to think about the correct definition of a mock behavior